docs/user-guide/resources.md

```diff
@@ -1,12 +1,12 @@
 # Resources
 
-The Openstack driver allows you to create Stacks in a target Openstack as part of a `Create` lifecycle transition (then remove the Stack with `Delete`). This is done by configuring the use of the driver on Create/Delete and by including Heat or Tosca templates in your Resource package. 
+The Openstack driver allows you to create new or adopt pre-existing Stacks in a target Openstack as part of a `Create` or `Adopt` lifecycle transition (then remove the Stack with `Delete`). This is done by configuring the use of the driver on Create/Adopt/Delete and by including Heat or Tosca templates in your Resource package. 
```

Comment:
Tosca templates are not supported for Adopt. Are they? If so, then I would make that clear.

Reply:
The Openstack driver does support TOSCA templates.

Reply:
I clarified that tosca/heat file only supplied in the case of `create`